tests/Basic/CookieTest.php

```diff
@@ -174,4 +174,20 @@ public function testSessionPersistsBetweenRequests()
         $node = $webAssert->elementExists('css', '#session-id');
         $this->assertEquals($newSessionId, $node->getText());
     }
+
+    public function testChangingCookieValueWhenTheFirstCookieWasSetByMinkAndNextFromHost()
+    {
+        $this->getSession()->setCookie('tc', 'original value');
```

Comment:
Please extract `$this->getSession()` into `$session` variable. Would make test much shorter and easier to read.